src/analyses/README.md

```diff
@@ -112,6 +112,9 @@ To be documented.
 
 To be documented.
 
-\subsection analyses-interval-analysist Integer interval analysis (interval_analysist) -- both an analysis and a transformation
+\subsection analyses-interval-analysis Integer interval analysis (\ref interval_analysis) -- both an analysis and a transformation
 
-To be documented.
+\ref interval_analysis interprets instructions of the input \ref goto_modelt
+over the \ref interval_domaint, evaluates variables for each program location
+(as intervals) and instruments the program with assertions representing the
```

Comment:
It seems that it is doing assumptions (the naming in the code is a bit confusing).